src/App.svelte

```diff
@@ -43,6 +41,6 @@
 
 <GlobalStyle />
 
-{#if leaderboard_loaded}
+{#if leaderboard}
```

Comment:
No need for a variable to track if it's loaded. The markup will be re-processes after `$:`.